### ISSUE TITLE
Filter Action: Save Attachement to local disk

### ISSUE THREAD

**Issue by techsurrusr** [RealRaven2000/quickFilters#269]
I noticed that in a previous version of [FiltaQuilla](https://github.com/RealRaven2000/quickFilters/issues/25#issuecomment-740755110), there were two filter actions: '**Detach Attachments To**' and '**Save Attachments To**'. It would be great if these could be re-implemented, and if they already exist, my apologies for missing the settings. These features would be very useful for allowing third-party applications to automatically process incoming attachments, such as invoices. Could you provide an update on the status of these features?

**Comment by RealRaven2000** [comment]
you mean these?

![image](https://github.com/user-attachments/assets/aaff50ed-202d-4c1e-b9dc-43302a5a1051)

They still exist, the screenshot is from Tb 131.0b5. Should we transfer this issue to the FiltaQuilla Github?

**Comment by techsurrusr** [comment]
No, my ask was specific for quickFilters which doesn't have these features, unless it's to be understod that FiltaQuilla should be used in combination with quickFilters?

**Comment by RealRaven2000** [comment]
> No, my ask was specific for quickFilters which doesn't have these features, unless it's to be understod that FiltaQuilla should be used in combination with quickFilters?

that's right. Use them together. quickFilters is not supposed to add new actions, it's completely different functionality. quickFilters is about filter management, creation, troubleshooting, backup and such things. FiltaQuilla adds more stuff to filtering. I don't want to add FiltaQuilla functionality into quickFilters it will get too complicated and dangerous because users will expect that attachments , reading body etc. work perfectly if they have a quickFilters Pro license. I am already doing too many unpaid support hours for FiltaQuilla so I am not prepared to add this here again and duplicate all the code.
